msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Customize Teams App Manifest in Teams Toolkit
+title: Customize App Manifest in Teams Toolkit
```

Comment:
Decapitalize "App Manifest" for consistency with other instances across docset

Reply:
Updated.